cmd/evm/internal/t8ntool/transition.go

```diff
@@ -247,7 +248,15 @@ func Transition(ctx *cli.Context) error {
 	}
 	// Sanity check, to not `panic` in state_transition
 	if chainConfig.IsLondon(big.NewInt(int64(prestate.Env.Number))) {
-		if prestate.Env.BaseFee == nil {
+		if prestate.Env.ParentBaseFee != nil {
```

Comment:
Nitpick: `basefee > parentBasefee`, precedence-wise IMO. 
So: only if prestate.Env.BaseFee is nill should we check prestate.Env.ParentBaseFee

Reply:
Updated.